packages/backend/src/index.ts

```diff
@@ -18,6 +18,9 @@
 
 import {createServer} from '@/server';
 
+// Set up the dotenv variables when running in development mode.
+require('dotenv')?.config();
```

Comment:
`?.` is not needed

This will run even in production, you need an if to check if NODE_ENV is `development`